examples/transform/src/main/java/org/apache/apex/examples/transform/CustomerInfo.java

```diff
@@ -72,11 +72,11 @@ public void setAddress(String address)
   @Override
   public String toString()
   {
-    return "CustomerInfo{" +
-      "customerId=" + customerId +
-      ", name='" + name + '\'' +
-      ", age=" + age +
-      ", address='" + address + '\'' +
+    return "{" +
```

Comment:
If this is just for test validation, why not verify the pojo instances instead of doing the toString conversion?

Reply:
fixed.

Reply:
revert the toString change?

Reply:
Done. Misunderstood the transform operator logic ( assumed it was serialising as a string ) Hence the original change. Reverted it back. Transform Operator should have been ideally a templatised instance.